openedx/features/enterprise_support/tests/test_api.py

```diff
@@ -671,9 +820,12 @@ def test_enterprise_learner_portal_message_cache_hit_no_customer(self, mock_lear
         self.assertIsNone(actual_result)
         self.assertFalse(mock_learner_data_from_db.called)
 
+    @ddt.data(True, False)
     @mock.patch('openedx.features.enterprise_support.api.get_enterprise_learner_data_from_db')
     @override_settings(ENTERPRISE_LEARNER_PORTAL_BASE_URL='http://localhost')
-    def test_enterprise_learner_portal_message_cache_hit_customer_exists(self, mock_learner_data_from_db):
+    def test_enterprise_learner_portal_message_cache_hit_customer_exists(
+            self, enable_learner_portal, mock_learner_data_from_db
```

Comment:
[NIT] put these on their own lines

Reply:
FWIW, since our code linters don't strictly require this, I use a sort of personal heuristic:
if it's a method signature for a test, I don't break each parameter onto its own line unless it's more than 2 parameters (other than self), or if the line containing all parameters is longer than the name of the method.